src/pear/script.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+## VIASH START
+## VIASh END
+
+[[ "$par_emperical_freqs" == "false" ]] && unset par_emperical_freqs
+[[ "$par_nbase" == "false" ]] && unset par_nbase
+
+ext=${par_forward_fastq##*.}
+if [[ "$ext" == "gz" ]]; then
+  gunzip $par_forward_fastq
+  gunzip $par_reverse_fastq
+  par_forward_fastq=${par_forward_fastq%.*}
+  par_reverse_fastq=${par_reverse_fastq%.*}
+fi
```

Comment:
These checks could be performed separately -- just in case

```suggestion
if [[ "${par_forward_fastq##*.}" == "gz" ]]; then
  gunzip $par_forward_fastq
  par_forward_fastq=${par_forward_fastq%.*}
fi
if [[ "${par_reverse_fastq##*.}" == "gz" ]]; then
  gunzip $par_reverse_fastq
  par_reverse_fastq=${par_reverse_fastq%.*}
fi
```